resources/fr.ftl

```diff
@@ -0,0 +1,61 @@
+auth = Veuillez utiliser le lien suivant pour vous authentifier sur votre compte Wikimedia : [Authenticate]({$url})
+
+auth_exists_in_server = Vous êtes déjà authentifié·ée sur ce serveur. Pas besoin de s'authentifier à nouveau.
+
+auth_to_server = Vous êtes déjà identifié·ée comme [{$name}](<{$url}>). Souhaitez-vous l'authentifier auprès du serveur ?
+yes = Oui
+no = Non
+
+auth_footer = Ce lien sera valide pendant 10 minutes.
+
+authreq_canceled = Authentification annulée.
+
+authreq_expired = L'authentification a expiré.
+
+authlog = {$mention} authentifié·ée comme [User:{$username}](<{$user_link}>) (id {$wmf_id})
```

Comment:
If you don't usually use the `User:` prefix, you can remove that for french.

Reply:
It's "Utilisateur" or "Utilisatrice"

Reply:
I leave User: which also works in URLs and is understandable. We sometimes replace words to avoid lengthening sentences. For example, there is administrateur and administratrice and we can use admin to make our lives easier.